website/docs/installing-podman-desktop/index.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 2
+title: Installing Podman Desktop
```

Comment:
We're in the docs for Podman Desktop. The nav title says Installing Podman Desktop, and just within the Windows tab we say 'Podman Desktop' 7 times and then have a podman-desktop tag. IMHO it's overkill and starting to hurt the readability. Could we remove PD from the navigation and a few of the more redundant instances?

Reply:
I added this to clarify that the topic is "Installing Podman Desktop" only, and not Podman, or any resource. I'll try to find a better way.